### DISCUSSION TITLE
Modify Site from Multiple PCs

### DISCUSSION THREAD

**Discussion by kirsch33** [GetPublii/Publii#1567] [Help]
Hi,

I have Publii on my laptop and it is setup to publish to my Github repo. My webserver also pulls down the latest updates from the same Github repo periodically so that part is all automated and works very nice.

However I would like to be able to store the work in progress website files that Publii uses for the user interface on the cloud as well so that I can edit the same website from any Publii instance (ie laptop vs desktop). However I cannot find a way to tell Publii to look for a website via a Github repo for example? 

Does anyone have any advice for this situation?

**Comment by johnwoltman** [comment]
I don't know how the github/git repo stuff works in Publii, but I'll tell you about my setup to maybe give you ideas:

- I moved my site to a folder on a cloud drive (in particular I use SyncThing)
- I told Publii where to look for the site.  To be honest I forgot how I did this because it was awhile ago.  Maybe a config file somewhere?
- I installed Publii on two computers, each with the same cloud drive.
- I made sure to only have Publii open on one computer at a time because I was concerned about messing up the site if two computers tried changing it.

Hope that helps

**Reply by candidexmedia** [reply]
> * I told Publii where to look for the site. To be honest I forgot how I did this because it was awhile ago. Maybe a config file somewhere?

It's in the [App Settings, under Files Location > Sites Location](https://getpublii.com/docs/publii-global-settings.html#fileslocation)

![](https://getpublii.com/docs/media/posts/19/files-location.svg)

**Reply by Sar6e** [reply]
I have a similar approach.  I leave the locations as default ~Documents/Publii.  I use SyncThing to synchronise my Documents folder across 

- Desktop
- Laptop
- NAS

So I can install/open Publii on any of my devices and it just works.  The NAS copy provides some protection from device failure or loss and get the files into my backup system for 'proper' backup.

**Comment by silbox-ch** [comment]
Hi @kirsch33 

Everything is in the documentation https://getpublii.com/docs/publii-on-multiple-computers-via-dropbox.html
